R/teal_slice.R

```diff
@@ -483,44 +512,40 @@ c.teal_slices <- function(...) {
   )
 }
 
+# store method for `teal_slices` object
+#' @param x `teal_slices` object
```

Comment:
This help page documents our own functions as well as methods for S3 generics. In the latter `teal_slice`(`s`) objects are passed the `x` argument and we cannot change that. Therefore, in our own functions these objects are passed to different arguments, e.g. `tss`. Please conform to this convention, otherwise we would have to say "`x`: for these functions this and for those functions that".

Reply:
duly noted
78ec8b086248a7452009bccce3f4e0f57fa1ced7